src/gui/widgets/scroll_text.hpp

```diff
@@ -110,6 +112,8 @@ class scroll_text : public scrollbar_container
 
 	point max_size_;
 
+	bool link_aware_;
```

Comment:
Are these needed as their own class members, or is it basically just forwarded through to one of the base classes?

Reply:
The latter, see `finalize_subclass` and the `build` method.